index.html

```diff
@@ -59,54 +59,148 @@
-                Donec hac dapibus arcu auctor gravida curae vel.
-              </p>
+      <div class="container">
+        <div class="myStory-wrapper">
+          <div class="myStory-section">
+            <div class="left-corner">
+              <h1>My story</h1>
+              <div>
+                <p>
+                  Lorem ipsum odor amet, consectetuer adipiscing elit. Netus
+                  habitant quam elit adipiscing tempor consectetur eget mattis.
+                  Donec hac dapibus arcu auctor gravida curae vel.
+                </p>
 
-              <p>
-                Commodo tellus praesent eleifend feugiat curae. Quisque eget
-                laoreet ridiculus suspendisse sollicitudin aliquam magna felis.
-                Vel laoreet potenti libero finibus sodales in tristique.
-              </p>
+                <p>
+                  Commodo tellus praesent eleifend feugiat curae. Quisque eget
+                  laoreet ridiculus suspendisse sollicitudin aliquam magna
+                  felis. Vel laoreet potenti libero finibus sodales in
+                  tristique.
+                </p>
 
-              <p>
-                Velit et dignissim torquent faucibus nunc. Egestas leo turpis
-                feugiat volutpat felis...
-              </p>
+                <p>
+                  Velit et dignissim torquent faucibus nunc. Egestas leo turpis
+                  feugiat volutpat felis...
+                </p>
+              </div>
+              <a href="mystory.html">
+                <button class="readmore-button">READ MORE</button>
+              </a>
             </div>
-            <a href="mystory.html">
-              <button class="readmore-button">READ MORE</button>
-            </a>
-          </div>
-          <div class="right-corner">
-            <div><img class="top-image" src="images/img4.jpg" alt="" /></div>
-            <div class="secondrow-images">
-              <img class="under-image" src="images/img5.jpg" alt="" />
-              <img class="under-image" src="images/img6.jpg" alt="" />
-              <img class="under-image" src="images/img7.jpg" alt="" />
+            <div class="right-corner">
+              <div><img class="top-image" src="images/img4.jpg" alt="" /></div>
+              <div class="secondrow-images">
+                <img class="under-image" src="images/img5.jpg" alt="" />
+                <img class="under-image" src="images/img6.jpg" alt="" />
+                <img class="under-image" src="images/img7.jpg" alt="" />
+              </div>
             </div>
           </div>
         </div>
-      </div>
 
-      <div class="blogs-wrapper">
-        <div>
-          <h1>LATEST FROM THE BLOG</h1>
-          <p>Our most recent city guides, iteneraries around the world</p>
-        </div>
-        <div></div>
-        <div>
-          <a href="blogs.html"><button></button></a>
+        <div class="blogs-section-container">
```

Comment:
move this inside .container

Reply:
I think it is already. Container has myStory-wrapper and blogs-section-container.